orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -948,6 +949,34 @@ class PipelineBatchPlanner {
   }
 };
 
+// verify pipeline config can load and gradient graph can construct.
+TEST(GradientGraphBuilderTest, TrainingSession_PipelineTransform_base) {
+  std::string filename_base = PIPELINE_MODEL_BASE;
+
+  auto load_gradient_graph = [](std::string& filename) {
+    auto config = MakeBasicTrainingConfig();
+
+    config.use_pipeline = true;
```

Comment:
>config.use_pipeline = true; [](start = 4, length = 27)

how do we insert Send Recv? I thought the config is supposed to have some info about where the cut point is

Reply:
Yes, that will be the next step -- adding the logic of graph cut into pipeline_transformer.

---
In reply to: [408351407](https://github.com/microsoft/onnxruntime/pull/3513#discussion_r408351407) [](ancestors = 408351407)